Models/ApiModel.php

```diff
@@ -0,0 +1,97 @@
+<?php
+declare(strict_types=1);
+
+namespace Models;
+
+/**
+ * @package Models
+ */
+class ApiModel extends DataModel
```

Comment:
Do you really need to extend DataModel here? Data model makes initialisation of PDO object with database connection. This class is not responsible for working with database. you don't need to extend it here